common/test/acceptance/tests/discussion/test_cohort_management.py

```diff
@@ -954,35 +953,6 @@ def test_verify_that_deselecting_the_final_child_deselects_category(self):
         # category should not be selected.
         self.assertFalse(self.cohort_management_page.is_category_selected())
 
-    def test_verify_that_correct_subset_of_category_being_selected_after_save(self):
```

Comment:
Note to reviewers: the actual code in this test didn't do what the description says. The description implies that an inline topic is being chosen with a category that has multiple inline topics, then it verifies that the overall category is not selected (because there is still at least one inline topic that is not selected). That seems like a useful test, except that the test course being used only has a single inline topic.

Rather than change the test course, I just deleted this test. We are likely changing the UI substantially soon anyway.